ebodac/src/main/java/org/motechproject/ebodac/template/PdfBasicTemplate.java

```diff
@@ -16,8 +16,8 @@
 
 public abstract class PdfBasicTemplate {
 
-    public static final Font TABLE_FONT = new Font(Font.FontFamily.HELVETICA, 8);
-    public static final Font HEADER_FONT = new Font(Font.FontFamily.HELVETICA, 9, Font.BOLD);
+    public static final Font TABLE_FONT = new Font(Font.FontFamily.HELVETICA, 6);
```

Comment:
I do not understand how is possible , when I merged  it was only the readme.md one file, and now appears more, I do not get it, could you just merge the readme.md ? the rest is not needed , those are old changes we discussed that were not needed, we just need the readme.md, please just merge the readme.md.